mmcv/utils/tensorrt_utils.py

```diff
@@ -0,0 +1,186 @@
+import ctypes
+import glob
+import os
+
+import tensorrt as trt
+
+
+def get_tensorrt_op_path():
+    wildcard = os.path.join(
+        os.path.abspath(os.path.dirname(os.path.dirname(__file__))),
+        '_ext_trt.*.so')
+
+    paths = glob.glob(wildcard)
+    if len(paths) > 0:
+        return paths[0]
+    else:
+        return None
+
+
+plugin_is_loaded = False
+
+
+def is_tensorrt_plugin_loaded():
+    global plugin_is_loaded
+    return plugin_is_loaded
+
+
+def load_tensorrt_plugin():
+    global plugin_is_loaded
+    if not plugin_is_loaded:
+        ctypes.CDLL(get_tensorrt_op_path())
+        plugin_is_loaded = True
+
+
+def onnx2trt(onnx_model,
```

Comment:
For core python function and classes, code comment should be added. Please take [nms](https://github.com/open-mmlab/mmcv/blob/d5241b70871c0664a1584989b58d7c507c240278/mmcv/ops/nms.py#L43) for an example.